CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 Change log
 -----------
 
+* Various fixes [Theodor]
```

Comment:
The Tegra BSP just makes use of the backward compatible variable, it is not specific to this BSP. Other BSP might still use this mechanism.